src/base/settingsstorage.cpp

```diff
@@ -26,7 +26,9 @@
  * but you are not obligated to do so. If you do not wish to do so, delete this
  * exception statement from your version.
  */
+#include "settingsstorage.h"
```

Comment:
Just caught this type of error in qBt. The error showed during the compilation of the moc files, in the very and of the compilation process. Should I have the paired include at the very top of the includes list, the error will not lead to recompiling half of the project after that (after fixing the header file).